### DISCUSSION TITLE
刚进空洞怎么给我传送到特雷森了

### DISCUSSION THREAD

**Discussion by SuzukaScenery** [share121/inter-knot#1920] [其他]
![Screenshot_2024-08-04-13-44-52-345_com komoe kmumamusumegp](https://github.com/user-attachments/assets/8292a515-b9e4-445c-beb0-c70d11008be9)
![1723435101499](https://github.com/user-attachments/assets/da805479-827f-4a47-9638-42dd674ded11)

**Comment by littlehorseaaa** [comment]
不是哥们，带我一个呗？

**Comment by Kamisato100** [comment]
异变，这是马儿跳异变啊！<img src="https://img-static.mihoyo.com/communityweb/upload/417976a3dacde790f947f8769d85d55c.png" alt="安详" width="75" >
